src/lib/components/icons/FolderCreate.svelte

```
@@ -0,0 +1,15 @@
+<script lang="ts">
+	export let className = 'size-4';
+	export let strokeWidth = '1.5';
```

ESLint

[error] Component has unused export property 'strokeWidth'. If it is for external reference only, please consider using `export const strokeWidth`(unused-export-let)

(svelte/valid-compile)